content/posts/about_inlining.md

```diff
@@ -28,22 +28,23 @@ Source : [inline specifier – cppreference.com][1]
 What we need to notice from that are two things:
```

Comment:
[Suggestion] If you want to specific any number of thoughts: use "1. 2. 3. x."; when you're not sure, you can write as "the followings". As this is a blog so this is not strictly as a document, but order always makes your passage more readable.